README.rst

```diff
@@ -56,14 +56,23 @@ or in batch mode on Windows, Linux, and even Mac OS.
 
 Documentation and issues
 ------------------------
-For more information, see the `Documentation <https://mapdl.docs.pyansys.com>`_ page.
-For some examples, see the `Examples gallery <https://mapdl.docs.pyansys.com/version/stable/examples/index.html>`_.
+Documentation for the latest stable release of PyMAPDL is hosted at `PyMechanical documentation
+<https://mapdl.docs.pyansys.com/version/stable/>`_.
 
-Feel free to post issues and other questions at `PyMAPDL Issues
-<https://github.com/pyansys/pymapdl/issues>`_.  This is the best place
-to post questions and code.
+In the upper right corner of the documentation's title bar, there is an option for switching from
+viewing the documentation for the latest stable release to viewing the documentation for the
+development version or previously released versions.
 
+You can also `view <https://cheatsheets.docs.pyansys.com/pymapdl_cheat_sheet.png>`_ or
+`download <https://cheatsheets.docs.pyansys.com/pymapdl_cheat_sheet.pdf>`_ the
+PyMAPDL cheat sheet. This one-page reference provides syntax rules and commands
+for using PyMAPDL. 
```

Comment:
We have the cheatsheet linked in [learning/downloads](https://mapdl.docs.pyansys.com/version/stable/getting_started/learning.html#downloads). I think that is the right place, and I will point to it in the docs.

Reply:
@germa89 Your suggestions are great. However, we are trying to establish one "common" place for users to look for this information. Are you OK with me adding the content I've proposed in this PR and keeping your link in the learning/downloads section.